.circleci/config.yml

```diff
@@ -36,34 +19,55 @@ jobs:
             - dependency-cache-{{ checksum "package-lock.json" }}
 
       - run:
-          name: Install cqlsh
-          # We need to edit DEFAULT_CQLVER defined in the cqlsh script
-          # to connect to the server without --cqlversion command line option
+          name: Docker login
           command: |
-            curl -O https://bootstrap.pypa.io/get-pip.py
-            sudo python get-pip.py
-            sudo pip install cqlsh
-            sudo sed -i "s/^DEFAULT_CQLVER = .*/DEFAULT_CQLVER = '3.4.4'/" /usr/local/bin/cqlsh
+            echo ${GHCR_PAT} | docker login https://ghcr.io -u ${GHCR_USERNAME} --password-stdin
 
       - run:
-          name: Wait for Cassandra
-          # dockerize command is included in circleci/openjdk:8-jdk image
-          command: dockerize -wait tcp://127.0.0.1:9042 -timeout 1m
+          name: Update docker and docker compose
+          command: |
+            sudo apt-get purge docker-ce containerd.io
```

Comment:
Just a question. Why not using the apt package?
I think it should be noted if there is a particular reason.

Reply:
The [docs here](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images) is stating the latest version, but after i debug it (ssh into the session), I found out that the docker version the image is using is a very old version and only supported up to docker-compose file of v3.2 which is not as they stated on that docs, so i have to manually update the docker version.